### PR TITLE
Simplify release instructions

### PR DESCRIPTION
**Pull request by vvagaytsev**
**What this PR does / why we need it**:
As we have already discussed, let's exclude the manual testing step from the release process.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: RELEASE_PROCESS.md [hunk 1]
```diff
@@ -25,44 +25,24 @@ and does the following:
...
-   - Checkout to the most recent pre-release branch, e.g. `1.2.3-0`, and cherry-pick the appropriate commits from `main`.
-   - Run `./scripts/release.ts prerelease` - it will generate a new pre-release `1.2.3-1`.
-   - Repeat the manual testing.
-5. If you’re ready to make a proper release, do the following:
-   - Checkout to the most recent pre-release branch, e.g. `1.2.3-1`.
-   - Remove all the `bump version...` commits. E.g. by using `git rebase -i <hash-before-first-version-bump>` and `drop`-ing the commits. In this case we drop `chore(release): bump version to 1.2.3-0` and `chore(release): bump version to v.1.2.3-1`.
-   - Run `./scripts/release.ts minor | patch`. This way, the version bump commits and changelog entries created by the pre-releases are omitted from the final history.
+2. Reset `latest-release` to `main` with `git reset --hard origin/main`
+3. Run `git log` to make sure that the latest commit is the expected one and there are no unwanted changes from `main` included in the release.
+4. Run `./scripts/release.ts minor | patch`. This way, the version bump commits and changelog entries created by the pre-releases are omitted from the final history.
```
Comment (Walther) [inline RELEASE_PROCESS.md:35]: let's make this easier to paste for the common case
```suggestion
4. Run `./scripts/release.ts patch`. This way, the version bump commits and changelog entries created by the pre-releases are omitted from the final history.
```

File: RELEASE_PROCESS.md [hunk 2]
```diff
@@ -25,44 +25,24 @@ and does the following:
...
-   - Automated release notes generation:
-     - Run `./scripts/draft-release-notes.ts <previous-tag> <current-tag>`, the filename with the draft notes will be printed in the console
-     - Open the draft file (it's named `release-notes-${version}-draft.md`, e.g. `release-notes-0.12.38-draft.md`) and resolve all suggested TODO items
-   - Old way of release notes generation (if the automated way fails for some reason):
-     - To generate a changelog for just that tag, run `git-chglog <previous-release-tag-name>..<tag-name>`
-     - To get a list of all contributors between releases, ordered by count, run: `./scripts/show-contributors.sh <previous-tag> <current-tag>`. Note that authors of squashed commits won't show up, so it might be good to do a quick sanity check on GitHub as well.
-     - Take the previous release notes for GitHub as a template and apply the necessary updates.
-     - Remember to put the list of features on top of the list of bug fixes in the changelog.
+  - Run `./scripts/draft-release-notes.ts <previous-tag> <current-tag>`, the filename with the draft notes will be printed in the console
```
Comment (Walther) [inline RELEASE_PROCESS.md:44]: sidenote: i wonder if we could make this a bit more automated, e.g. not having to specify the tags manually?
we might also need that for the more automated release process in the future

**Review by Walther — APPROVED** [review]
Looks good, excellent work 🎉 
Much easier to follow now.

Approved, with one small nit an one even smaller sidenote, no blockers.

File: RELEASE_PROCESS.md (hunk 2 above)
Comment (vvagaytsev) [reply RELEASE_PROCESS.md:44]: We'll see what the release automation tools can offer. As far as I know, GitHub can do the changelog generation quite well.

Comment (Walther) [reply RELEASE_PROCESS.md:44]: sounds good!

**Pull request merged**